edu.wpi.first.wpilib.plugins.cpp/resources/templates/command-based/TimedCommand.cpp

```diff
@@ -0,0 +1,27 @@
+#include "$classname.h"
+
+$classname::$classname(double timeout) : TimedCommand(timeout) {
+	// Use Requires() here to declare subsystem dependencies
+	// eg. Requires(chassis);
+}
+
+// Called just before this Command runs the first time
+void $classname::Initialize() {
+
+}
+
+// Called repeatedly when this Command is scheduled to run
+void $classname::Execute() {
+
+}
+
+// Called once after command times out
+void $classname::End() {
+
+}
+
+// Called when another command which requires one or more of the same
+// subsystems is scheduled to run
+void $classname::Interrupted() {
```

Comment:
Since `TimedCommand` overrides `Command`, `Interrupted()` is implemented to call `End()`, which is usually the desired behavior. Having it in the template could help remind teams that `Interrupted` is available, but it should call its parent's `Interrupted()` method by default.